draft-scenarios/clash/king_of_the_hill.tex

```diff
@@ -0,0 +1,117 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{King of the Hill}{\images/kingofthehill.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} LAAMAKALA
+
+\textit{In the battle for the Hill, there is no honor — only victory or death. The weak will fall, the strong will fight, and only one will rule.}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+This Scenario plays out over 8 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2 -- 4
+
+\textbf{Starting Resources:} 15 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item A Few of \silver\ with the lowest Recruitment cost.
+  \item 1 random Neutral \bronze\ Unit.
+\end{itemize}
+
+\textbf{Town Buildings:}
+\begin{itemize}
+  \item \bronze\ Dwelling
+  \item City Hall
+  \item \svg{building_special_tent} Building
+\end{itemize}
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):
+
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item 2P × Far (II-III) Map Tiles
+  \item 2P × Near (IV-V) Map Tiles
+  \item 1 × Center Map Tile. The center Field is ``The Hill''
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+The game ends after Round 8.
```

Comment:
This does not sound like a win con. The goal is to score as many VPs as you can, right?

Reply:
I think removing is fine.